### ISSUE TITLE
[Bug] namespace Blazored.Modal not found

### ISSUE THREAD

**Issue by omarceloribeiro** [Blazored/Modal#566]
**Describe the bug**
I just installed the package 7.3.1 in a .net 7.0 blazor webassembly, and the namespace Blazored.Modal can not be found.
dowgraded to 7.2.0 and it works.
the error occurrs with the version 7.3.0 and 7.3.1

**To Reproduce**
1. create a blazor web assembly in .net 7.0
2. install blazored.modal
3. try to add the using Blazored.Modal in the program.cs

**Expected behavior**
the namespace is recognized.

**Screenshots**

**Hosting Model (is this issue happening with a certain hosting model?):**
 - Blazor WebAssembly Client ex: MyApp.Client
 - .net core hosted ex: MyApp.Server

**Additional context**
dowgraded to 7.2.0 and it works.
the error occurrs with the version 7.3.0 and 7.3.1

`<PropertyGroup>
  <TargetFramework>net7.0</TargetFramework>
  <Nullable>enable</Nullable>
  <ImplicitUsings>enable</ImplicitUsings>
  <ServiceWorkerAssetsManifest>service-worker-assets.js</ServiceWorkerAssetsManifest>
</PropertyGroup>

<ItemGroup>
  <PackageReference Include="Blazored.Modal" Version="7.2.0" />
`

**Comment by zootius** [comment]
Seconded - this happens for me in my Blazor WASM project too. Has been happily working with Blazored.Modal 7.1 - but with any version after 7.2, it reports Blazored.Modal namespace not found.

**Comment by MilesGibson** [comment]
me as well!  I had to downgrade...

**Comment by chrissainty** [comment]
I can't repro this issue on my machines. Can anyone provide a link to a repo showing the issue?

**Comment by abbottmw** [comment]
Are there plans to make it work for .net 7 or  going to be 8.0 minimum moving forward?

**Comment by chrissainty** [comment]
@abbottmw I don't think that I'll add .NET 7 support. It goes out of support in May so people should be moving to .NET 8.

**Comment by ChrisASearles** [comment]
Had the same problem, any chance of updating the versioning scheme to match .NET's? I didn't expect a breaking change like that in my .NET 7 project going from one 7.x minor version to another.
